ui/components-react/src/components-react/editors/EditorContainer.tsx

```diff
@@ -130,78 +145,104 @@ export class EditorContainer extends React.PureComponent<EditorContainerProps> {
     }
 
     return clonedNode;
-  }
+  };
 
-  private _handleEditorBlur = (_e: React.FocusEvent) => {
+  /** Event Handlers
+   * @internal
+   */
+  const handleCancel = () => onCancel();
+  const handleClick = (e: React.MouseEvent) => e.stopPropagation();
+  const handleContainerBlur = (e: React.FocusEvent) => e.stopPropagation();
+  const handleEditorCommit = (args: PropertyUpdatedArgs): void =>
+    void commit(args);
+
+  const handleContainerCommit = async (): Promise<void> => {
+    const newValue = editorRef && (await editorRef.current?.getPropertyValue());
     // istanbul ignore else
-    if (
-      !this.props.ignoreEditorBlur &&
-      this._propertyEditor &&
-      this._propertyEditor.containerHandlesBlur
-    )
-      void this._handleContainerCommit();
+    if (newValue !== undefined) {
+      void commit({
+        propertyRecord,
+        newValue,
+      });
+    }
   };
 
-  private _handleContainerBlur = (e: React.FocusEvent) => {
-    e.stopPropagation();
+  const onPressEscape = (): void => {
+    // istanbul ignore else
+    if (propertyEditor && propertyEditor.containerHandlesEscape) handleCancel();
```

Comment:
This seems to match the previous handler.
I can't seem to reproduce the #630 here https://itwin.github.io/appui/storybook/?path=/docs/components-editors--docs
What was the cause and what changed to fix it?

Reply:
I'm not sure to be honest. I wasn't able to recreate it either but I made the switch that was recommended

Reply:
Should we update the PR description then if we are not sure that it fixes the #630?
Were you able to reproduce that issue before your changes?

Reply:
I changed the PR description to address that. I did try the suggested change but it wasn't successful